website/docs/cdktf/concepts/resources.mdx

```diff
@@ -370,6 +370,66 @@ import imports.kubernetes.namespace.NamespaceMetadata;
 
 </CodeTabs>
 
+## Refactoring & Renaming Resources
```

Comment:
We have a guide on refactoring, we should update it using the move block. (We should also write one for import, if we have time)

Reply:
Great catch, if anything that section logically belongs in our refactoring guide.

[Moved the bulk of the section](https://github.com/hashicorp/terraform-cdk/pull/3152/commits/9a29db9c7fc3c16d444e7fc727101b56ac829af0) to the refactoring guide. I then have `resources.mdx` mention the functionality and refer to the refactoring guide for more info.